dev/build

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-go build -ldflags "-X main.version=dev-$(date +%F-%T)" -o /usr/local/bin/yeydev ./src
\ No newline at end of file
+go build -ldflags "-X main.version=dev-$(date +%F-%T)" -o /usr/local/bin/yey ./src
```

Comment:
not for this PR bit gonna mention it here: let's consider using a version.txt that we can embed. It will be easier for CI versioning